### PR TITLE
use `__about__` for package metadata on docs and setup

### PR DESCRIPTION
**Pull request by tony**
This way, only one location needs to be updated for package versions/naming on docs/conf.py and setup.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 from setuptools import setup
 
+about = {}
+with open("django_activeurl/__about__.py") as fp:
```
Comment (hellysmile) [inline setup.py]: io.open with explicit encoding and mode please

File: docs/conf.py [hunk 2]
```diff
@@ -1,242 +1,66 @@
...
-# unit titles (such as .. function::).
-#add_module_names = True
+import django_activeurl  # NOQA
 
-# If true, sectionauthor and moduleauthor directives will be shown in the
-# output. They are ignored by default.
-#show_authors = False
+# package data
+about = {}
+with open("../django_activeurl/__about__.py") as fp:
```
Comment (hellysmile) [inline docs/conf.py]: io.open with explicit encoding and mode please

**Review by hellysmile — CHANGES_REQUESTED** [review]
Can You please change file opening way

**Comment by tony** [comment]
Updated,  ``mode='r'`` is default behavior in ``io.open``. Do you want it added explicitly?

**Comment by hellysmile** [comment]
just encoding is perfectly fine! thnx!

**Pull request merged**